src/index.js

```diff
@@ -61,10 +61,19 @@ const formatOptions = ({steps, ...otherOptions}) => ({
   steps: (steps || ['welcome','document','face','complete']).map(formatStep)
 })
 
+const deprecationWarnings = ({steps}) => {
```

Comment:
nice ;)

With this neat solution you can even add it to the setOptions, but it's minor.